src/templates/en/2020/methodology.html

```diff
@@ -360,39 +364,76 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
+        The 2020 Web Almanac kicked off in <a href="https://twitter.com/rick_viscomi/status/1273135952848977920">June 2020</a>, later than the 2019 timeline due to the unrest related to COVID-19 and the social justice protests. These and other events of 2020 were an undercurrent throughout the entire production process and put a lot of additional strain on our contributors beyond the stresses of a fast-paced project like this.
+      </p>
+
+      <p>
+        As we <a href="https://almanac.httparchive.org/en/2019/methodology#brainstorming">stated in last year's Methodology</a>:
+      </p>
+
+      <blockquote>
+        One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </blockquote>
+
+      <p>
+        To that end, this year we've made systematic changes to the way that we seek and select authors:
+
+        <ul>
+          <li>
+            Previous authors were specifically discouraged from writing again to make room for different perspectives.
+          </li>
+          <li>
+            Everyone endorsing 2020 authors were asked to be especially conscious not to nominate people who all look or think alike.
+          </li>
+          <li>
+            Many 2019 authors were Google employees and this year we tried to get a greater balance of perspectives from the broader web community. We believe that the voices in the Web Almanac should be a reflection of the community itself, and not skewed towards any specific company to avoid creating echo chambers.
+          </li>
+          <li>
+            The project leads reviewed all of the author nominations and made an effort to select authors who will bring new perspectives and amplify the voices of underrepresented groups in the community.
+          </li>
+        </ul>
+
+        We hope to iterate on this process in the future to ensure that the Web Almanac is a more diverse and inclusive project with contributors from all backgrounds.
       </p>
 
       <p>
-        We spent July 2020 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
+         Finally, in July 2020, after rounds of brainstorming and nominations, 22 chapters were solidified and we formed content teams for each chapter tasked with writing, reviewing, and analysis.
```

Comment:
Not sure we need the "finally" here? Reads like part of a list when it's not.

```suggestion
        In July 2020, after rounds of brainstorming and nominations, 22 chapters were solidified and we formed content teams for each chapter tasked with writing, reviewing, and analysis.
```

Reply:
I felt like this needed a transition word to move from the inclusion process to team formation.